source/fundamentals/builders/sort.txt

```diff
@@ -11,3 +11,346 @@ Sorts Builders
    :class: singlecol
 
 .. _sorts-builders:
+
+Overview
+--------
+
+In this guide, we show you how to use **builders** to specify
+**sort criteria** for your queries.
+
+Sort criteria are the rules MongoDB uses to sort your data. Some examples of sort criteria are:
+
+* Smallest number to largest number
+* Earliest time of day to latest time of day
+* Alphabetical order by first name 
+
+Builders are classes provided by the MongoDB Java driver that allow you to use
+the builder pattern to interact with your MongoDB instance.
+The builder pattern allows you to create complex
+immutable objects separately from their representation. The
+:doc:`builders </fundamentals/builders/>`
+provided by the MongoDB Java driver can improve the readability of your code.
+
+You should read this guide if you would like to:
+
+* Use the builder pattern to specify sort criteria for your queries.
+* Learn how to perform ascending sorts, descending sorts, and combining sorts.
+* Sort on the text score of :manual:`$text </reference/operator/query/text/>` 
+  queries.
+
+If you are new to sorting in the
+MongoDB Java driver, consider reading our
+:doc:`guide on sorting </fundamentals/crud/read-operations/sort/>`
+for a general introduction to sorting with the MongoDB Java driver.
+
+The Sorts Class
+---------------
+
+The :java-docs:`Sorts </apidocs/mongodb-driver-core/com/mongodb/client/model/Sorts.html>`
+class is the sort criteria builder provided by the MongoDB Java Driver.
+``Sorts`` provides static factory methods for all
+sort criteria operators supported by MongoDB. Each method returns
+an instance of the
+:java-docs:`Bson </apidocs/bson/org/bson/conversions/Bson.html>`
+type. 
+
+.. _docs_java_fundamentals_builders_sort_sort_example:
+
+The following examples show you how to use the static factory
+methods provided by the ``Sorts`` class to sort your queries.
+The examples use a sample collection ``sort_example`` that contains
+the following documents:
+
+.. code-block:: json
+
+   {"_id": 1, "letter": "c", "food": "coffee with milk"}
+   {"_id": 3, "letter": "a", "food": "maple syrup"}
+   {"_id": 4, "letter": "b", "food": "coffee with sugar"}
+   {"_id": 5, "letter": "a", "food": "milk and cookies"}
+   {"_id": 2, "letter": "a", "food": "donuts and coffee"}
+   {"_id": 6, "letter": "c", "food": "maple donut"}
+
+
+Sorting Direction
+-----------------
+
+The ``Sorts`` class provides two methods for specifying the sort order of data
+that does not contain any duplicate values. Data can be sorted from smallest to largest 
```

Comment:
The second part of the sentence could use explanation. This can be interpreted as "you cannot use these methods if there's duplicate data" which isn't the message I think you're trying to convey.

Reply:
Thanks for the note. I see now how that sentence could be read in that unintended way. I am using the term "sort direction" to remove discussion of duplicate values from this sentence entirely. Fix: https://github.com/mongodb/docs-java/pull/59/commits/714f1a252b15bea083f5c054f08ee85c3a17dc8f